<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1252</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TerraView 5.0</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1252</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="m_addOGRLayer_act"/>
    <addaction name="m_addGDALLayer_act"/>
    <addaction name="separator"/>
    <addaction name="m_fileNewProject"/>
    <addaction name="m_fileOpenProject"/>
    <addaction name="m_fileSaveProject"/>
    <addaction name="m_fileSaveProjectAs"/>
    <addaction name="separator"/>
    <addaction name="m_filePrint"/>
    <addaction name="m_filePrintPreview"/>
    <addaction name="separator"/>
    <addaction name="m_fileImportProject"/>
    <addaction name="m_fileExportProject"/>
    <addaction name="separator"/>
    <addaction name="actionRecent_Projects"/>
    <addaction name="separator"/>
    <addaction name="m_fileExit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="m_editUndo"/>
    <addaction name="m_editRedo"/>
    <addaction name="separator"/>
    <addaction name="m_editCut"/>
    <addaction name="m_editCopy"/>
    <addaction name="m_editPaste"/>
    <addaction name="separator"/>
    <addaction name="m_editSelectAll"/>
    <addaction name="m_editClear"/>
    <addaction name="separator"/>
    <addaction name="m_editFind"/>
    <addaction name="m_editReplace"/>
    <addaction name="separator"/>
    <addaction name="m_editDataSourceManager"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Toolbars"/>
    <addaction name="separator"/>
    <addaction name="m_viewLayerExplorer"/>
    <addaction name="m_viewMapDisplay"/>
    <addaction name="m_viewTableData"/>
    <addaction name="m_viewRasterVisual"/>
    <addaction name="separator"/>
    <addaction name="m_viewFullScreen"/>
    <addaction name="separator"/>
    <addaction name="m_viewRefresh"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="m_settingsPreferences"/>
   </widget>
   <widget class="QMenu" name="menu_Plugins">
    <property name="title">
     <string>&amp;Plugins</string>
    </property>
    <addaction name="m_pluginManager"/>
    <addaction name="m_pluginBuilder"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="m_helpContents"/>
    <addaction name="m_helpUpdate"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_Plugins"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="m_tools_TB">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_actionPan"/>
   <addaction name="m_actionZoom_area"/>
   <addaction name="m_actionPointing"/>
   <addaction name="separator"/>
   <addaction name="m_distance_act"/>
   <addaction name="m_angle_act"/>
   <addaction name="m_area_act"/>
   <addaction name="separator"/>
   <addaction name="m_drawlayers_act"/>
  </widget>
  <widget class="QToolBar" name="m_viewComp_tb">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_viewLayerExplorer"/>
   <addaction name="m_viewMapDisplay"/>
   <addaction name="m_viewTableData"/>
   <addaction name="m_viewRasterVisual"/>
  </widget>
  <action name="m_fileNewProject">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>New project...</string>
   </property>
   <property name="toolTip">
    <string>Creates a new TerraView project...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_fileOpenProject">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open project...</string>
   </property>
   <property name="toolTip">
    <string>Opens an existing TerraView project...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_fileSaveProject">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save project...</string>
   </property>
   <property name="toolTip">
    <string>Saves the current TerraView project.</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_actionPan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pan...</string>
   </property>
   <property name="toolTip">
    <string>Set current tool to pan...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_actionZoom_area">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Zoom area...</string>
   </property>
   <property name="toolTip">
    <string>Sets the current tool to zoom area.</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_actionPointing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Pointing...</string>
   </property>
   <property name="toolTip">
    <string>Sets the current tool to pointing.</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_viewLayerExplorer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Layer Explorer</string>
   </property>
   <property name="toolTip">
    <string>Shows or hides the layer explorer.</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_viewMapDisplay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Map Display</string>
   </property>
   <property name="toolTip">
    <string>Shows or hides the map display.</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_viewTableData">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Table Data</string>
   </property>
   <property name="toolTip">
    <string>Shows or hide the table with tabular data.</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_addOGRLayer_act">
   <property name="text">
    <string>Add OGR Layer...</string>
   </property>
   <property name="toolTip">
    <string>Creates a new OGR layer.</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_drawlayers_act">
   <property name="text">
    <string>Draw layers...</string>
   </property>
   <property name="toolTip">
    <string>Draws the visible layers at main display.</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_distance_act">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Distance measure tool...</string>
   </property>
   <property name="toolTip">
    <string>Sets the distance measure tool.</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_angle_act">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Angle measure tool...</string>
   </property>
   <property name="toolTip">
    <string>Sets the angle measure tool.</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_area_act">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Area measure tool...</string>
   </property>
   <property name="toolTip">
    <string>Sets the area measure tool</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_addGDALLayer_act">
   <property name="text">
    <string>Add GDAL Layer...</string>
   </property>
   <property name="toolTip">
    <string>Creates a new GDAL layer.</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_viewRasterVisual">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Raster Visual</string>
   </property>
   <property name="toolTip">
    <string>Shows or hides the raster visual window.</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_fileSaveProjectAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Project &amp;As...</string>
   </property>
   <property name="toolTip">
    <string>Save project as...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_fileImportProject">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Import Project...</string>
   </property>
   <property name="toolTip">
    <string>Imports a project.</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_fileExportProject">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Export Project...</string>
   </property>
   <property name="toolTip">
    <string>Exports project.</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_filePrint">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Print...</string>
   </property>
   <property name="toolTip">
    <string>Print map.</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_filePrintPreview">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Print Pre&amp;view...</string>
   </property>
   <property name="toolTip">
    <string>Print preview.</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_editUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Undo...</string>
   </property>
   <property name="toolTip">
    <string>Undo last operation.</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_editRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Redo...</string>
   </property>
   <property name="toolTip">
    <string>Redo last operation</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_editCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cu&amp;t...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_editCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Copy...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_editPaste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Paste...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_editSelectAll">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Select &amp;All...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_editClear">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>C&amp;lear...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_editFind">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Find...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_editReplace">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>R&amp;eplace...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_viewFullScreen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>F&amp;ull Screen...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_viewRefresh">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Refresh...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_settingsPreferences">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Preferences...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_pluginManager">
   <property name="text">
    <string>&amp;Manage Plugins...</string>
   </property>
   <property name="toolTip">
    <string>Shows plugins manager dialiog.</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_pluginBuilder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Build a New Plugin...</string>
   </property>
   <property name="toolTip">
    <string>Shows the dialog to create a new plugin.</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_helpContents">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;View Help...</string>
   </property>
   <property name="toolTip">
    <string>Shows help dialog.</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_helpUpdate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Update...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRecent_Projects">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Recent Projects</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_fileExit">
   <property name="text">
    <string>E&amp;xit...</string>
   </property>
   <property name="toolTip">
    <string>Shutdown system.</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_Toolbars">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Toolbars...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="m_editDataSourceManager">
   <property name="text">
    <string>&amp;Manage Data Sources...</string>
   </property>
   <property name="toolTip">
    <string>Shows data source manager.</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_addOGRLayer_act</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>addOGRLayer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_drawlayers_act</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>drawLayers()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_actionPan</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>setPanTool(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_actionZoom_area</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>setZoomAreaTool(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_angle_act</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>setAngleTool(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_area_act</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>setAreaMeasureTool(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_distance_act</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>setDistanceTool(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_addGDALLayer_act</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>addGDALLayer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_fileExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>625</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_editDataSourceManager</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openDSrcManager()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>625</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pluginManager</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openPluginsManager()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>625</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addOGRLayer()</slot>
  <slot>drawLayers()</slot>
  <slot>setPanTool(bool)</slot>
  <slot>setZoomAreaTool(bool)</slot>
  <slot>setDistanceTool(bool)</slot>
  <slot>setAreaMeasureTool(bool)</slot>
  <slot>setAngleTool(bool)</slot>
  <slot>addGDALLayer()</slot>
  <slot>openPluginsManager()</slot>
  <slot>openDSrcManager()</slot>
 </slots>
</ui>
